src/Resolvers/Microsoft.DotNet.MSBuildSdkResolver/MSBuildSdkResolver.cs

```diff
@@ -66,17 +74,29 @@ public override SdkResult Resolve(SdkReference sdkReference, SdkResolverContext
             List<string> warnings = null;
             CachingWorkloadResolver workloadResolver = null;
 
+            using var logger = new ResolverLogger(sdkReference, _shouldLog);
+
+            logger.LogMessage($"Attempting to resolve MSBuild SDK {sdkReference.Name}");
```

Comment:
You'll want to use `logger.LogMessage(string format, params object[] args)` and only call `string.Format()` if logging is enabled, otherwise these strings are going to be allocated for nothing right?